src/haddock/modules/analysis/clustfcc/__init__.py

```diff
@@ -250,4 +263,25 @@ def _run(self) -> None:
             log.warning("No clusters were found")
             self.output_models = models_to_cluster  # type: ignore
 
+        # Draw the matrix
+        if self.params['plot_matrix']:
+            # Obtain final models indices
+            final_order_idx, labels = [], []
+            for pdb in self.output_models:
+                final_order_idx.append(models_to_cluster.index(pdb))
```

Comment:
If I understand correctly this means that there's no re-ordering of models prior to plotting, right?

Reply:
Yes, the ordering is taken from the self.output_models, for agreements with next modules.
At this stage, they are ordered by cluster sizes, as found in `cluster.out` file minus the ones excluded from the parameters.